.github/workflows/pypi-release.yml

```diff
@@ -0,0 +1,33 @@
+name: Prepare release
```

Comment:
Maybe rename this as https://github.com/metabrainz/picard/blob/master/.github/workflows/github-release.yml#L1 already defines a workflow with the same name.

Reply:
Thanks, I forgot to rename it. Fixed